documentation/intro-dylan/source/conditions.rst

```diff
@@ -147,8 +147,8 @@ depending on the color of the sky.
     end block;
 
 Many programs need to dispose of resources or perform other cleanup work when
-exiting a block. The block may contain optional ``afterwards`` and ``cleanup``
-clauses. Neither affects the block's return value. The ``afterwards`` clause
+exiting a block. The block may contain optional :drm:`afterwards <block>` and ``cleanup``
+clauses. Neither affects the block's return value. The :drm:`afterwards <block>` clause
```

Comment:
Maybe we can just add an index entry for "afterwards" to the DRM instead?

Reply:
I agree.

Reply:
`afterwards` is not in DRM index and the only reference is in the `Other Statement macros` with `block`. Where should `afterwards` point to?

Reply:
That sounds right, yes.  Same as block but it's own entry.